doc/api/inspector.md

```diff
@@ -183,15 +183,25 @@ changes:
 * `method` {string}
 * `params` {Object}
 * `callback` {Function}
+* Returns: {Promise|undefined}
 
 Posts a message to the inspector back-end. `callback` will be notified when
 a response is received. `callback` is a function that accepts two optional
 arguments: error and message-specific result.
 
+If the `callback` is not passed, then it'll return a Promise result.
```

Comment:
maybe this should be done via util.promisify instead of implicit behavior?

Reply:
Well, as a lot of other functions were converted to Promises I thought this might be something good to the user land. WDYT?

Reply:
Are there a lot that do it in this way, with an optional callback?

Reply:
To be clear, what I’m suggesting is adding a custom promisify implementation if needed, and *not* returning a promise from the actual function - so that users can `util.promisify(session.post)` explicitly.

Reply:
hey! I just changed to /promises api. Would you check it now?